Libplanet/Net/Swarm.cs

```diff
@@ -545,6 +541,40 @@ IEnumerator IEnumerable.GetEnumerator()
                 cancellationToken);
         }
 
+        /// <summary>
+        /// Synchronizes the blocks with other registerd <see cref="Peer"/>s.
+        /// </summary>
+        /// <param name="blockChain">
+        /// A <see cref="BlockChain{T}"/> instance to synchronize.
+        /// </param>
+        /// <param name="progress">
+        /// An instance that receives progress updates for block downloads.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// A cancellation token used to propagate notification that this
+        /// operation should be canceled.
+        /// </param>
+        /// <typeparam name="T">An <see cref="IAction"/> type. It should match
+        /// to <see cref="BlockChain{T}"/>'s type parameter.</typeparam>
+        /// <returns>
+        /// No object or value is returned by this method when it completes.
+        /// </returns>
```

Comment:
If we omit `<returns>` tag at all does the compiler warns of it?

Reply:
Yes. linter says '[SA1615](http://www.stylecop.soyuz5.com/SA1615.html): Element return value shuld be documented.'